src/Illuminate/Collections/Collection.php

```diff
@@ -1628,7 +1628,7 @@ public function offsetSet($key, $value)
     /**
      * Unset the item at a given offset.
      *
-     * @param  string  $key
+     * @param  string|int  $key
```

Comment:
This should be `mixed` actually: https://www.php.net/manual/en/arrayaccess.offsetunset.php